azure-pipelines.release.yml

```diff
@@ -0,0 +1,109 @@
+pr: none
+trigger: none
+
+variables:
+  - group: 'Github and NPM secrets'
+
+pool:
+  vmImage: 'Ubuntu 16.04'
+
+steps:
+  - task: NodeTool@0
+    inputs:
+      versionSpec: '10.x'
+    displayName: 'Install Node.js'
+
+  - script: |
+      git config user.name "UI Fabric Build"
+      git config user.email "fabrictactical@service.microsoft.com"
+      git remote set-url origin https://$(githubUser):$(githubPAT)@github.com/OfficeDev/office-ui-fabric-react.git
+    displayName: Authenticate git for pushes
+
+  - script: |
+      node common/scripts/install-run-rush.js install --bypass-policy
+    displayName: rush install
+
+  - script: |
+      npm run generate-version-files
+    displayName: npm run generate-version-files
+
+  - script: |
+      node common/scripts/install-run-rush.js build --production --verbose
+    displayName: rush build (Create production build)
+
+  - script: |
+      mkdir -p $(Build.ArtifactStagingDirectory)/api
+      cp packages/office-ui-fabric-react/dist/office-ui-fabric-react.api.json $(Build.ArtifactStagingDirectory)/api
+      cp packages/merge-styles/dist/merge-styles.api.json $(Build.ArtifactStagingDirectory)/api
+      cp packages/styling/dist/styling.api.json $(Build.ArtifactStagingDirectory)/api
+      cp packages/utilities/dist/utilities.api.json $(Build.ArtifactStagingDirectory)/api
+    displayName: Copy api.json files to artifact staging directory
+
+  - task: PublishBuildArtifacts@1
+    inputs:
+      pathtoPublish: $(Build.ArtifactStagingDirectory)/api
+      artifactName: 'api-json'
+      publishLocation: 'Container'
+    displayName: 'Publish Artifact: api.json'
+
+  - task: PublishBuildArtifacts@1
+    inputs:
+      pathtoPublish: apps/fabric-website/dist
+      artifactName: 'fabric-website'
+      publishLocation: 'Container'
+    displayName: 'Publish Artifact: Fabric Website'
+
+  - task: PublishBuildArtifacts@1
+    inputs:
+      pathtoPublish: apps/fabric-website/index.html
+      artifactName: 'fabric-website-index'
+      publishLocation: 'Container'
+    displayName: 'Publish Artifact: Fabric Website index.html'
+
+  - task: PublishBuildArtifacts@1
+    inputs:
+      pathtoPublish: packages/office-ui-fabric-react/dist
+      artifactName: 'fabric'
+      publishLocation: 'Container'
+    displayName: 'Publish Artifact: Fabric'
+
+  - script: |
+      node ./common/scripts/install-run-rush.js publish -a -p -b master -n $(npmToken) --add-commit-details --set-access-level public
+    displayName: 'Publish Change Requests'
+
+  - script: |
+      node scripts/updateReleaseNotes.js --token=$(githubPAT) --apply --debug
+    displayName: 'Update github release notes'
+
+  - script: |
+      oufrVersion=`grep '"version": ' packages/office-ui-fabric-react/package.json | sed 's/\s*\"version\": \"\(.*\)\".*/\1/'`
```

Comment:
Same issue here with ensuring that errors fail the build

Reply:
found a better one liner...

Reply:
> found a better one liner...

Nice :D.. this is relevant to my interests.